receiver/src/main/java/net/azisaba/ballotbox/receiver/fetcher/impl/MySQLVoteFetcher.java

```diff
@@ -92,10 +88,13 @@ public boolean connect() {
           currentIndex = Long.max(currentIndex, id);
         }
 
+        statement.close();
+        resultSet.close();
```

Comment:
ResultSet is automatically closed by calling `statement.close();` so this is no-op